.yarnrc.yml

```diff
@@ -9,4 +10,5 @@ plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-version.cjs
     spec: "@yarnpkg/plugin-version"
 
-yarnPath: .yarn/releases/yarn-berry.cjs
+yarnPath: .yarn/releases/yarn-3.3.0.cjs
+checksumBehavior: "update"
```

Comment:
I can confirm that this can be removed in Yarn 4: https://github.com/jubeira/balancer-v2-monorepo/pull/9.

I didn't find any other sensible way of dealing with this so far, and I'm not sure it's worth investing more time in it.
I'd either go with this flag, or move to Yarn 4.

Reply:
Is there a reason to use 3 over 4?

Reply:
Yarn 4 latest version is `4.0.0-rc29`, so technically it's still not released yet. 

But other than that both seem to work. Besides the checksum fix, apparently in version 4 they've bundled the plugins we use, so there's no need to declare them in the `.yarnrc.yml` file.

Reply:
Update: the current verson is rc31 and it also works.

According to https://github.com/yarnpkg/berry/discussions/4895#discussioncomment-3726438, it should be fine to use an RC version. I don't think they'll actually break anything we use when they release 4.0.0.

Reply:
Updated to rc42 in https://github.com/balancer/balancer-v2-monorepo/pull/2020/commits/c4b55babbc476a96a39f8c1f77d0964f5a987b13, and it still works.